templates/components/_footer.html.twig

```diff
@@ -0,0 +1,124 @@
+                <li><a href="https://www.youtube.com/channel/UCJw8np695wqWOaKVhFjkRyg" target="_blank" rel="noopener noreferrer"><i class="fa fa-youtube-square"></i></a></li>
+                <li><a href="https://medium.com/@enmarchefr" target="_blank" rel="noopener noreferrer"><i class="fa fa-medium"></i></a></li>
+                <li><a href="https://www.linkedin.com/company/en-marche" target="_blank" rel="noopener noreferrer"><i class="fa fa-linkedin-square"></i></a></li>
+                <li><a href="https://www.snapchat.com/add/enmarchefr" target="_blank" rel="noopener noreferrer"><i class="fa fa-snapchat-ghost"></i></a></li>
+            </ul>
+        </div>
+        <div class="l__row l__row--h-stretch l__row--top b__nudge--bottom-large l__row--wrap l__mobile--col">
+            <ul class="list__links list__links--col list__links--no-decor">
+                <li>
+                    <h4>
+                        <a href="{{ path('page_le_mouvement') }}" class="text--bold text--medium-small">Le Mouvement</a>
+                    </h4>
+                </li>
+                <li>
+                    <a href="{{ path('page_le_mouvement') }}">Qui sommes-nous ?</a>
+                </li>
+                <li>
+                    <a href="{{ path('page_le_mouvement_notre_organisation') }}">Notre organisation</a>
+                </li>
+                <li>
+                    <a href="{{ path('page_emmanuel_macron') }}">Emmanuel Macron</a>
+                </li>
+                <li>
+                    <a href="{{ path('legislatives_homepage') }}" target="_blank" rel="noopener noreferrer">
+                        Nos députés
+                    </a>
+                </li>
+                {% if not campaign_expired %}
+                    <li>
+                        <a href="{{ path('page_le_mouvement_les_comites') }}">Comités</a>
+                    </li>
+                    <li>
+                        <a href="{{ path('app_search_events') }}">Événements</a>
+                    </li>
+                    <li>
+                        <a href="{{ path('page_le_mouvement_devenez_benevole') }}">Devenez bénévole</a>
+                    </li>
+                {% endif %}
+                <li>
+                    <a rel="noopener noreferrer" href="https://storage.googleapis.com/en-marche-prod/documents/adherents/1-charte-et-manifeste/charte_des_valeurs.pdf">Charte des valeurs</a>
+                </li>
+            </ul>
+            <ul class="list__links list__links--col list__links--no-decor">
+                <li>
+                    <h4>
+                        <a href="{{ path('page_campus') }}" class="text--bold text--medium-small">Campus</a>
+                    </h4>
+                </li>
+                <li>
+                    <a href="{{ path('page_campus') }}">Toutes les ressources</a>
```

Comment:
Same link as above, intended?

Reply:
same, can you confirm it is intended @marineb ?